Terminal Autostart Functionality

Notenlish opened the pull request:
I thought I'd just be done with adding the command to add to .bashrc and be done with it but no there is a slight bug with anifetch when its ran on terminal autostart. 

When terminal starts(no animation displayed yet), it gives this error:
```
stty: 'standard input': Inappropriate ioctl for device
stty: 'standard input': Inappropriate ioctly for device
```
after terminal has started playing the animation and I do ctrl c, it gives this:
stty: 'standard input': Inappropriate ioctly for device

this should be fixed with a startup flag spesifically for terminal autostarts.

Also maybe another flag for making the anifetch/neofetch output on top/bottom of the animation rather than to the side maybe? And another flag for potentially making the anifetch/neofetch output invisible maybe?

Notenlish committed:
Update README.md

Gallophostrix commented:
I'm using it currently with the `anifetch video.mp4 [...]` command directly in the `.bashrc` file to launch it after the terminal start, which works fine

But I'm not sure that's what you're talking about!

Notenlish commented:
> I'm using it currently with the `anifetch video.mp4 [...]` command directly in the `.bashrc` file to launch it after the terminal start, which works fine
> 
> But I'm not sure that's what you're talking about!

I added the command to my .bashrc with the `exec` command so maybe that's why?

CurrentscoobS commented:
I am currently trying to utilize terminal autostart, where I'm using zsh and receiving the same error you found regarding the inappropriate ioctl for device. I have had no luck in getting this to work within my zshrc, what did you change in order to resolve this issue in particular?

Notenlish commented:
The solution would be to add a flag called --startup and dont run stty if the flag exists. but we don't have a pr for it rn

Notenlish committed:
added startup functionality

Notenlish commented:
@CurrentscoobS can you try installing this pr onto your machine via pipx and changing your config to include: `anifetch example.mp4`

CurrentscoobS commented:
@Notenlish I force installed anifetch via pipx, loaded the new comment into my bashrc and it sort of just seems to hang indefinitely, I end up having to Ctrl+C to end the process. But the inappropriate ioctl doesn't show up on bash currently! Although, I also tried loading this in my zshrc which is my default, and it still shows the ioctl error.

Notenlish commented:
> I end up having to Ctrl+C to end the process

That's normal, the animation repeats indefinitely until you do ctrl+c

> I also tried loading this in my zshrc which is my default, and it still shows the ioctl error.
did you put it at the end of the .zshrc

honestly, idk what would be the solution. Maybe try using `~/.zprofile` instead of `~/.zshrc`

Notenlish commented:
> it still shows the ioctl error.

I just installed zshell, it doesnt give any issues to me, maybe it has to do with your terminal or shell configuration?

Here is my config:
```zsh
# Set up the prompt

autoload -Uz promptinit
promptinit
prompt adam1

setopt histignorealldups sharehistory

# Use emacs keybindings even if our EDITOR is set to vi
bindkey -e

# Keep 1000 lines of history within the shell and save it to ~/.zsh_history:
HISTSIZE=1000
SAVEHIST=1000
HISTFILE=~/.zsh_history

# Use modern completion system
autoload -Uz compinit
compinit

zstyle ':completion:*' auto-description 'specify: %d'
zstyle ':completion:*' completer _expand _complete _correct _approximate
zstyle ':completion:*' format 'Completing %d'
zstyle ':completion:*' group-name ''
zstyle ':completion:*' menu select=2
eval "$(dircolors -b)"
zstyle ':completion:*:default' list-colors ${(s.:.)LS_COLORS}
zstyle ':completion:*' list-colors ''
zstyle ':completion:*' list-prompt %SAt %p: Hit TAB for more, or the character to insert%s
zstyle ':completion:*' matcher-list '' 'm:{a-z}={A-Z}' 'm:{a-zA-Z}={A-Za-z}' 'r:|[._-]=* r:|=* l:|=*'
zstyle ':completion:*' menu select=long
zstyle ':completion:*' select-prompt %SScrolling active: current selection at %p%s
zstyle ':completion:*' use-compctl false
zstyle ':completion:*' verbose true

zstyle ':completion:*:*:kill:*:processes' list-colors '=(#b) #([0-9]#)*=0=01;31'
zstyle ':completion:*:kill:*' command 'ps -u $USER -o pid,%cpu,tty,cputime,cmd'

# My changes
export PATH="$HOME/.local/bin:$PATH"
anifetch example.mp4
```

Notenlish committed:
skipping frames added

its not related in any way to this branch but im too lazy to create a new branch.

Notenlish commented:
added a potential fix for #27

Notenlish committed:
Update README.md

Notenlish committed:
forgot to remove this

Notenlish committed:
fix text of -ff --fastfetch help
